openslides/mediafiles/static/js/mediafiles/resources.js

```diff
@@ -93,6 +103,49 @@ angular.module('OpenSlidesApp.mediafiles.resources', [
     }
 ])
 
+.factory('Logos', [
```

Comment:
This factory is only loosly coupled to the Logo ressource so I would prefere moving this code to another module, maybe `logo.js`.